### PR TITLE
test(scheduling): add skeleton for allow empty selector settings

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: [longhorn/longhorn#4826](https://github.com/longhorn/longhorn/issues/4826)

### CODE REVIEW

**Review by roger-ryao — DISMISSED** [review]
LGTM

File: manager/integration/tests/test_scheduling.py [hunk 1]
```diff
@@ -1722,3 +1722,67 @@ def finalizer():
...
+    - Set allow-empty-node-selector-volume to `false`
+
+    When
+    - Create a Volume with 3 replicas without tag
+
+    Then
+    - All replicas can not be scheduled to the nodes
+
+    When
+    - Set allow-empty-node-selector-volume to `true`
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_scheduling.py:1750]: Before setting `allow-empty-node-selector-volume` to `true`,  can we add one more step of removing the `AVAIL` tag from one of the node and as a result volume replica should get scheduled on that node without tag. 

Purpose of adding the test step is to ensure `allow-empty-node-selector-volume` setting is respecting tags correctly and just not blindly overruling the schedule criteria.

File: manager/integration/tests/test_scheduling.py [hunk 2]
```diff
@@ -1722,3 +1722,67 @@ def finalizer():
...
+    - Set allow-empty-disk-selector-volume to `false`
+
+    When
+    - Create a Volume with 3 replicas without tag
+
+    Then
+    - All replicas can not be scheduled to the disks on the nodes
+
+    When
+    - Set allow-empty-disk-selector-volume to `true`
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_scheduling.py:1782]: Same as above comment, can we add a step to remove tag from one of the disks and replica should get scheduled on that?

**Review by khushboo-rancher — CHANGES_REQUESTED** [review]

File: manager/integration/tests/test_scheduling.py (hunk 1 above)
Comment (ChanYiLin) [reply manager/integration/tests/test_scheduling.py]: Fixed, thanks!

File: manager/integration/tests/test_scheduling.py (hunk 2 above)
Comment (ChanYiLin) [reply manager/integration/tests/test_scheduling.py]: Fixed, thanks!

**Comment by ChanYiLin** [comment]
> This pull request is now in conflicts. Could you fix it @ChanYiLin? 🙏

Conflicts resolved

**Review by khushboo-rancher — APPROVED** [review]
